### PR TITLE
Lambda Layer Update: Layer ARNs to the latest published NOV-2022

### PR DESCRIPTION
**Pull request by vasireddy99**
**Description:** 

Updates the layer.tf and template.yml file to have the latest Lambda Layer ARNs for the layers released as part of NOV 2022 release.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: nodejs/sample-apps/aws-sdk/deploy/wrapper/layer_arm64.tf [hunk 1]
```diff
@@ -1,14 +1,20 @@
...
-    "ap-southeast-1" = "arn:aws:lambda:ap-southeast-1:901920570463:layer:aws-otel-nodejs-arm64-ver-1-7-0:1"
-    "ap-southeast-2" = "arn:aws:lambda:ap-southeast-2:901920570463:layer:aws-otel-nodejs-arm64-ver-1-7-0:1"
-    "eu-central-1"   = "arn:aws:lambda:eu-central-1:901920570463:layer:aws-otel-nodejs-arm64-ver-1-7-0:1"
-    "eu-west-1"      = "arn:aws:lambda:eu-west-1:901920570463:layer:aws-otel-nodejs-arm64-ver-1-7-0:1"
-    "eu-west-2"      = "arn:aws:lambda:eu-west-2:901920570463:layer:aws-otel-nodejs-arm64-ver-1-7-0:1"
-    "us-east-1"      = "arn:aws:lambda:us-east-1:901920570463:layer:aws-otel-nodejs-arm64-ver-1-7-0:1"
-    "us-east-2"      = "arn:aws:lambda:us-east-2:901920570463:layer:aws-otel-nodejs-arm64-ver-1-7-0:1"
-    "us-west-2"      = "arn:aws:lambda:us-west-2:901920570463:layer:aws-otel-nodejs-arm64-ver-1-7-0:1"
+    "ap-northeast-1" = "arn:aws:lambda:ap-northeast-1:901920570463:layer:aws-otel-nodejs-arm64-ver-1-7-0:2"
+    "ap-northeast-2" = "arn:aws:lambda:ap-northeast-2:901920570463:layer:aws-otel-nodejs-arm64-ver-1-7-0:1"
```
Comment (bryan-aguilar) [inline nodejs/sample-apps/aws-sdk/deploy/wrapper/layer_arm64.tf:4]: Why do some of these have different version(?) numbers? `:1` vs `:2`.

File: java/sample-apps/okhttp/deploy/wrapper/layer_arm64.tf [hunk 2]
```diff
@@ -1,14 +1,21 @@
...
-    "eu-west-2"      = "arn:aws:lambda:eu-west-2:901920570463:layer:aws-otel-java-wrapper-arm64-ver-1-18-0:1"
-    "us-east-1"      = "arn:aws:lambda:us-east-1:901920570463:layer:aws-otel-java-wrapper-arm64-ver-1-18-0:1"
-    "us-east-2"      = "arn:aws:lambda:us-east-2:901920570463:layer:aws-otel-java-wrapper-arm64-ver-1-18-0:1"
-    "us-west-2"      = "arn:aws:lambda:us-west-2:901920570463:layer:aws-otel-java-wrapper-arm64-ver-1-18-0:1"
+    "ap-northeast-1" = "arn:aws:lambda:ap-northeast-1:901920570463:layer:aws-otel-java-wrapper-arm64-ver-1-19-0:1"
+    "ap-northeast-2" = "arn:aws:lambda:ap-northeast-2:901920570463:layer:aws-otel-java-wrapper-arm64-ver-1-19-0:1"
+    "ap-south-1"     = "arn:aws:lambda:ap-south-1:901920570463:layer:aws-otel-java-wrapper-arm64-ver-1-19-0:1"
+    "ap-southeast-1" = "arn:aws:lambda:ap-southeast-1:901920570463:layer:aws-otel-java-wrapper-arm64-ver-1-19-0:1"
+    "ap-southeast-2" = "arn:aws:lambda:ap-southeast-2:901920570463:layer:aws-otel-java-wrapper-arm64-ver-1-19-0:1"
+    "ca-central-1"   = "arn:aws:lambda:ca-central-1:901920570463:layer:aws-otel-java-wrapper-arm64-ver-1-19-0:2"
```
Comment (Aneurysm9) [inline java/sample-apps/okhttp/deploy/wrapper/layer_arm64.tf:9]: Why is this particular region on a different layer version than the rest?

File: nodejs/sample-apps/aws-sdk/deploy/wrapper/layer_arm64.tf (hunk 1 above)
Comment (Aneurysm9) [inline nodejs/sample-apps/aws-sdk/deploy/wrapper/layer_arm64.tf:4]:
```suggestion
    "ap-northeast-2" = "arn:aws:lambda:ap-northeast-2:901920570463:layer:aws-otel-nodejs-arm64-ver-1-7-0:2"
```

Double check all of these.

File: java/sample-apps/okhttp/deploy/wrapper/layer_arm64.tf (hunk 2 above)
Comment (vasireddy99) [reply java/sample-apps/okhttp/deploy/wrapper/layer_arm64.tf:9]: I have copied this from the `tf` layer generated in the [workflow](https://github.com/aws-observability/aws-otel-lambda/actions/runs/3425866372/jobs/5707088914#step:8:1), ca-central-1 region has published the layer version 2,
 The initial workflow has encountered an error and cancelled. [version 1 ](https://github.com/aws-observability/aws-otel-lambda/actions/runs/3425767316/jobs/5706875157#step:9:81)was copied/published (not a public layer yet) to prod at this time for this region. when retriggered then, the correct layer came out as version 2.

**Comment by vasireddy99** [comment]
We are publishing the following new regions for `arm64` architecture. Since nodejs layer version/name has not changed for this release. these new regions have published as version 1 and the rest of exisitng regions has  published as version 2, since version 1 for this existing regions are published in previous release that had v0.22.0 ADOT collector. 

image

**Review by bryan-aguilar — APPROVED** [review]

**Pull request merged**